R/backend-postgres.R

```diff
@@ -229,6 +229,36 @@ sql_expr_matches.PqConnection <- sql_expr_matches.PostgreSQLConnection
 
 # DBI methods ------------------------------------------------------------------
 
+#' @export
+db_write_table.PostgreSQLConnection <- function(con, table, types, values,
```

Comment:
I think maybe it might be best to put this in a separate file, to make it more clear that its for backward compatibility with RPostgresSQL. Then the test could be specific to RPostgreSQL (no longer using the `test_register_con()`) and exercise the full pipeline (i.e. copy data in, write simple query, and retrieve data).